prince-of-versions/src/main/java/co/infinum/princeofversions/PrinceOfVersions.java

```diff
@@ -49,11 +49,11 @@ public PrinceOfVersions(Context context) {
     }
 
     @VisibleForTesting
-    PrinceOfVersions(Storage storage, Executor callbackExecutor, ApplicationConfiguration appConfig) {
+    public PrinceOfVersions(Storage storage, Executor callbackExecutor, ApplicationConfiguration appConfig) {
         this(createDefaultParser(), createDefaultVersionParser(), storage, callbackExecutor, appConfig);
     }
 
-    private PrinceOfVersions(ConfigurationParser configurationParser, VersionParser versionParser, Storage storage,
+    public PrinceOfVersions(ConfigurationParser configurationParser, VersionParser versionParser, Storage storage,
```

Comment:
Why do we need public without @VisibleForTesting here?

Reply:
I am not sure about this one. I can't remember why did I actually left that or even why I even changed anything. Should I return it to private or add @VisibleForTesting?

Reply:
Depending on if that constructor is used from testing or not